Cargo.toml

```diff
@@ -24,7 +25,7 @@ cssparser-macros = {path = "./macros", version = "0.6"}
 dtoa-short = "0.3"
 itoa = "1.0"
 matches = "0.1"
-phf = {version = ">=0.8,<=0.11", features = ["macros"]}
+phf = {version = "~0.9", features = ["macros"]}
```

Comment:
Bumping seems fine, but can we just use 0.11 here now?

Reply:
phf 0.11 has an MSRV of 1.60.

Reply:
That's fine, we can have the lockfile with `0.10` or so.